src/store/saga/index.js

```diff
@@ -0,0 +1,80 @@
+import {
+  getMoviesListRequest,
+  getTrendingMoviesListRequest,
+  getUpcomingMoviesListRequest,
+  getTvShows,
+  getPeopleRequest,
+  search,
+} from "../requests";
+import {
+  setMovieListAction,
+  setTrendingMoviesAction,
+  setUpcomingMoviesAction,
+  selection,
+  setTvShowsAction,
+  setPeopleAction,
+  setSearch,
+} from "../actions/movieAction";
+import { takeEvery, all, call, put, debounce } from "@redux-saga/core/effects";
```

Comment:
Absolute imports should always be on top.

Reply:
done